astropy/units/format/base.py

```diff
@@ -133,37 +122,49 @@ def to_string(
         ValueError
             If ``fraction`` is not recognized.
         """
+        string_components = cls._to_string_helper(unit, fraction)
+        if isinstance(string_components, str):
+            return string_components
+        if fraction is True or fraction == "inline":
+            return cls._format_inline_fraction(*string_components)
+        raise ValueError(
+            f"format {cls.name!r} only supports 'inline' fractions, not {fraction=!r}."
+        )
+
+    @classmethod
+    def _to_string_helper(
```

Comment:
I've realized that a much better name for this function would be
```suggestion
    def _to_string_fraction_helper(
```
but I'm not pushing this change to GitHub until @mhvk has confirmed that he is happy with the way the code works, or we've come to an agreement about an alternative.

Reply:
I like that new suggested name!

Reply:
Agreed. It's much clearer, esp when encountering code like. is being discussed on line 125.